src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -261,5 +261,32 @@ public async Task<ActionResult> CreateRelatedOrganisationAsync(Guid id, [FromBod
 
             return NoContent();
         }
+
+        [Authorize(Policy = PolicyName.CanAccessOrganisations)]
+        [HttpDelete]
+        [Route("{id}/related-organisations/{relatedOrganisationId}")]
+        public async Task<ActionResult> DeleteRelatedOrganisationAsync(Guid id, Guid relatedOrganisationId)
+        {
+            var organisation = await organisationRepository.GetByIdWithRelatedOrganisationsAsync(id);
+
+            if (organisation is null)
+            {
+                return NotFound();
+            }
+
+            if (!organisation.RelatedOrganisations.Any(ro => ro.OrganisationId == relatedOrganisationId))
+            {
+                return NoContent();
+            }
+
+            var relatedOrganisation = organisation.RelatedOrganisations.Single(ro => ro.OrganisationId == relatedOrganisationId);
```

Comment:
If we update this to `SingleOrDefault`, I think we should be able to remove the `if` statement above without requiring any additional changes (`Remove` will accept a null reference).

Reply:
that's true, but i'm not 100% sure how EF will handle it.
should be fine, but gona double check